prow/cmd/checkconfig/main.go

```diff
@@ -295,16 +295,18 @@ func validateTideRequirements(cfg *config.Config, pcfg *plugins.Configuration) e
 
 	// Now actually execute the checks we just configured.
 	var validationErrs []error
-	for _, pluginConfig := range configs {
-		err := ensureValidConfiguration(
-			pluginConfig.plugin,
-			pluginConfig.label,
-			pluginConfig.matcher.verb,
-			pluginConfig.config,
-			overallTideConfig,
-			enabledOrgReposForPlugin(pcfg, pluginConfig.plugin, pluginConfig.external),
-		)
-		validationErrs = append(validationErrs, err)
+	if pcfg != nil {
```

Comment:
This seems wrong - why bother calling to validate tide requirements if we don't execute anything?

Reply:
@amwat

Reply:
lol yeah... you can just skip this entire `validateTideRequirements` check

Reply:
cleaned up in https://github.com/kubernetes/test-infra/pull/11416 :)